OJP/OJP_Trips.xsd

```diff
@@ -878,6 +895,172 @@
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="Speed" default="average" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference about the speed/short duration of the trip.</xs:documentation>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="important"/>
+						<xs:enumeration value="average"/>
+						<xs:enumeration value="unimportant"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="Comfort" default="average" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference about the comfort/convenience of the trip, including few changes, short transfers and convenient vehicle types. Reflect USER PREFERENCE from Transmodel</xs:documentation>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="premium"/>
+						<xs:enumeration value="average"/>
+						<xs:enumeration value="basic"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="Sportiness" default="average" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's endurance and fitness for sportive, self-powered trip legs (walk, cycle, etc.).</xs:documentation>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="high"/>
+						<xs:enumeration value="average"/>
+						<xs:enumeration value="low"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="EnvironmentalSafety" default="average" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference about climate/environmental-friendliness of the trip (option 'low' might have no effect).</xs:documentation>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="high"/>
+						<xs:enumeration value="average"/>
+						<xs:enumeration value="low"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="WeatherProtection" type="xs:boolean" default="false" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference not be exposed to eventual weather hazards, thus, to avoid walks and micromobility.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="ExtraSafe" type="xs:boolean" default="false" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference for an extra high level of safety.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="ExtraReliable" type="xs:boolean" default="false" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference for an extra high reliability (low probabilty of delays, cancellations, etc.).</xs:documentation>
+				</xs:annotation>
```

Comment:
_[This extra text could be added to give more hints about the meaning and how it should be implemented]_
If true, modes known for their (un)reliability may be avoided/preferred, and extra time added for transfers.
This may depend on the actual, local or time of day situation,
based on punctuality statics, traffic jam statistics or rush hours.
E.g. taxis in a given city might be known to be unreliable during at 8-10 and 16-19 hours, otherwise reliable.